lib/videoroom/meeting.ex

```diff
@@ -6,27 +6,47 @@ defmodule Videoroom.Meeting do
 
   require Logger
 
-  alias Jellyfish.ServerMessage.PeerCrashed
-  alias Jellyfish.ServerMessage.PeerDisconnected
-  alias Jellyfish.ServerMessage.RoomCrashed
+  alias Jellyfish.{Peer, Room}
+  alias Jellyfish.ServerMessage.{PeerConnected, PeerCrashed, PeerDisconnected, RoomCrashed}
 
-  alias Jellyfish.Room
   alias Videoroom.RoomRegistry
 
+  @enforce_keys [
+    :name,
+    :client,
+    :notifier,
+    :room_id,
+    :peer_timers,
+    :peer_timeout
+  ]
+
+  defstruct @enforce_keys
```

Comment:
Question: 
Should GenServer's state be defined as a struct? If so, should the struct be defined as a nested module inside `Meeting` or can the `Meeting` itself be this struct?

Reply:
I think it would be better to make it a nested module, otherwise the types and the struct become a part of API of meeting, while it's internal